neat/tools/fittools/compartmentfitter.py

```diff
@@ -1014,15 +1135,15 @@ def fitModel(self, loc_arg, alpha_inds=[0], use_all_chans_for_passive=True,
         loc_arg: list of locations or string (see documentation of
                 :func:`MorphTree._convertLocArgToLocs` for details)
             The compartment locations
-        alpha_inds: list of ints (optional, default to ``[0]``)
+        alpha_inds: list of ints
```

Comment:
why remove the documentation for default values?

Reply:
it can be seen from the function definition